.github/workflows/checks.yml

```diff
@@ -54,15 +54,35 @@ jobs:
         with:
           test-results: test.json
 
-  e2e-test:
+  e2e-quick-start:
     strategy:
       matrix:
         provider:
           - Docker
+          # Uncomment below once we have the ability to run e2e tests on AWS from GHA.
+          # - AWS
```

Comment:
or Nutanix. was wondering how are we planning to pass the setup details of either aws or nutanix

Reply:
GHA Secrets should be enough I think.

Reply:
Yes these will need to be environment variables specfified either via GHA secrets when running in GHA or just normal env vars (via a local envrc perhaps) for local development. These should be looked up via the e2e variables though as a consistent way to interact with variables either from e2e config file or from the environment.